lingetic-nextjs-frontend/app/components/questions/SourceToTargetTranslation/Question.tsx

```
@@ -0,0 +1,42 @@
+    disabled,
+}: QuestionBoxProps) {
+    return (
+        <div className="flex flex-col gap-4">
+            <div className="text-lg">
+                {question.sourceText}
+            </div>
+            <div>
+                <input
+                    type="text"
+                    value={value}
+                    autoFocus
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)